vnext/Scripts/copyRNLibraries.js

```diff
@@ -95,6 +95,7 @@ exports.copyRNLibraries = baseDir => {
     },
```

Comment:
What's the dev workflow for consuming RNTester, running integration tests, anything else not published/copied?

Reply:
I haven't worked this out yet.  I think I would prefer running an install step and generate a react-native-installation for each package in the repo, so that it works as closely as possible to outside the repo, but this might get messy.

---
In reply to: [405211993](https://github.com/microsoft/react-native-windows/pull/4522#discussion_r405211993) [](ancestors = 405211993)